ArmRobot/Assets/Scripts/RandomizerPositionObject.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class RandomizerPositionObject : MonoBehaviour
+{
+
+
+    private GameObject gameObjectSeen;
+    float robotMinReach = 0.2f;
+    float robotMaxReach = 0.5f;
```

Comment:
There are identical `robotMinReach` and `robotMaxReach` properties on the `DomainRandomizer` class. A user should not have to change the exact same property on multiple scripts. Please keep these values in only one place.